pandamium_datapack/data/pandamium/functions/main_loop.mcfunction

```diff
@@ -1,6 +1,7 @@
 tp @e[type=minecraft:wither] 0 -1000 0
 
-tp @a[nbt={Health:0f}] 0 1000 0
+scoreboard players set @e[type=player] time_since_death 1
```

Comment:
Why have you used `@e[type=player]` rather than `@a`?

Reply:
`@e[type=player]` only selects alive players